BLE_BatteryLevel/shields/TARGET_CORDIO_BLUENRG.lib

```diff
@@ -1 +1 @@
-https://github.com/ARMmbed/cordio-ble-x-nucleo-idb0xa1/#811f3fea7aa8083c0bbf378e1b51a8b131d7efcc
+https://github.com/ARMmbed/cordio-ble-x-nucleo-idb0xa1/#08be35e6f91c1f7632bbae69372d1ac836980bda
```

Comment:
The reference to mbed-os must be updated if the latest cord driver is used.

Reply:
[Done](https://github.com/ARMmbed/mbed-os-example-ble/compare/54a30552020233cc3f1d4599e6f10ccac7af01b5..3f4c7d378db7c611b390b14a3bd5d5b04decd68c).